### ISSUE TITLE
508 issue with Incremental Practice Exam Topic 2

### ISSUE THREAD

**Issue by KRGould** [Section508Coordinators/DHSA-TT-200-A#344]
## Issue Type (put a capital X between the brackets to indicate issue type)
- [ ] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [ ] Incorrect content
- [X ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description

Section 508 Trusted Tester Incremental Practice Exam>Topic 2 Auto-Play... Exam contains elements that create a FLASHING hazard under 508 guidelines

## Location
https://exams-dev1-c.section508testing.net/dhsv5/dhscourse/Lesson1/exams/web-incremental/testPages/index.php?studentEmail=mr.kr.gould@gmail.com&SCOInstanceID=3251&examId=3

## Actual Behavior / Observation
Flashing element, Red on white. text

## Expected Behavior / Recommendation
Description of flashing content in lieu of actual content.

## Steps to Reproduce the Issue (if applicable)
In Topic 3, Lesson 1>Evaluating Content, the statement is made “Out of caution, this course only describes pages with flashing or blinking content and will not present any live examples.”

However,  Topic 2: Auto-playing….Content> Auto-Playing and Auto-Updating Content – Incremental Practical Exam, the page launched when “test this page” is engaged, contains an element id=”physicalHealthHeader” that flashes, toggling a visibility:hidden style element. ADDITIONALLY, I was unable to find a warning that the page contained that type of element.  

If the concern is to prevent flashing material from inadvertently triggering a photosensitive disorder, please reconsider removing the live example(s) in other modules.

**Comment by drewnielson** [comment]
@mikebarklage, although the "Physical Health On Hold" text in incremental exam 2 is well outside of the 3 flashes per second threshold that would classify as "flashing" per WCAG, it is still true that some people with high sensitivity could still be affected. For the failure instances where the "Physical Health On Hold" text fails for Test ID 2.B, can we make the text scroll across the screen instead of blink?

@Tonylayton1 and @mikebarklage - I can't remember off the top of my head, but we might also want to take a similar approach if we have similar content in the Practice Exam and/or Final, i.e., scroll instead of blink.

**Comment by mikebarklage** [comment]
I have updated this on Dev.

**Comment by Tonylayton1** [comment]
verified on dev

**Comment by drewnielson** [comment]
verified on dev
